AudioOutput.h

```diff
@@ -106,6 +106,8 @@ struct StereoOutput {
   static inline StereoOutput fromNBit(uint8_t bits, int16_t l, int16_t r) { return StereoOutput(SCALE_AUDIO(l, bits), SCALE_AUDIO(r, bits)); }
   /** @see MonoOutput::fromNBit(), stereo variant, 32 bit overload */
   static inline StereoOutput fromNBit(uint8_t bits, int32_t l, int32_t r) { return StereoOutput(SCALE_AUDIO(l, bits), SCALE_AUDIO(r, bits)); }
+  /** @see MonoOutput::fromNBit(), stereo variant, native int bit overload */
+  static inline StereoOutput fromNBit(uint8_t bits, int l, int r) { return StereoOutput(SCALE_AUDIO(l, bits), SCALE_AUDIO(r, bits)); }  
```

Comment:
Could you try
```
template<typename T> static inline StereoOutput fromNBit(uint8_t bits, T l, T r) { return StereoOutput(SCALE_AUDIO(l, bits), SCALE_AUDIO(r, bits)); }
```
instead? If this works, it could replace both existing versions of the function, rather than adding a third.

Reply:
Will try.

Reply:
Seems to work. See latest commit.